core/domain/feedback_domain.py

```diff
@@ -137,3 +137,18 @@ def to_dict(self):
             'state_content': self.state_content
         }
 
+
+class FeedbackMessageReferences(object):
```

Comment:
This is just for a single reference, right? So the class name should be FeedbackMessageReference